app/models/document.rb

```diff
@@ -3,6 +3,7 @@ class Document < ApplicationRecord
 
   belongs_to :owning_document, class_name: "Document", optional: true
   has_many :editions
+  has_one :link_set, primary_key: "content_id", foreign_key: "content_id", inverse_of: :document
```

Comment:
I've often wondered what the point of `LinkSet` actually is, since it appears to hold no more data than the `Document` itself.  But I guess that's something for another time.